CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.14.3] - 2022-11-08
+### Added
+- Add variable to set custom environment variables for the Hive Metastore
```

Comment:
Shouldn't this be a minor bump?

Reply:
so this should be `6.15.0` ?

Reply:
I would say so, yes. It's not a patch, you're just adding new functionality that is backwards compatible.